ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/deprovision/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-deprovision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
```

Comment:
this chain no need as we reuse `cucushift-installer-rehearse-vsphere-ipi-deprovision`